test/recipes/20-test_cli_fips.t

```diff
@@ -26,25 +26,14 @@ use platform;
 my $no_check = disabled('fips-securitychecks');
 plan skip_all => "Test only supported in a fips build with security checks"
     if disabled("fips") || disabled("fips-securitychecks");
-plan tests => 13;
+plan tests => 11;
 
 my $fipsmodule = bldtop_file('providers', platform->dso('fips'));
 my $fipsconf = srctop_file("test", "fips-and-base.cnf");
 my $defaultconf = srctop_file("test", "default.cnf");
 my $tbs_data = $fipsmodule;
 my $bogus_data = $fipsconf;
 
-# output a fipsmodule.cnf file containing mac data
-ok(run(app(['openssl', 'fipsinstall', '-out', 'fipsmodule.cnf',
-            '-module', $fipsmodule, ])),
-   "fipsinstall");
-
-# verify the $fipsconf file
-ok(run(app(['openssl', 'fipsinstall', '-in', 'fipsmodule.cnf', '-module', $fipsmodule,
-            '-verify'])),
```

Comment:
Should the verification be kept?

Reply:
Sure.  But in that case, as a separate sanity check

Reply:
Added